src/csv-writer.py

```diff
@@ -6,7 +6,7 @@
 def export_json_file_to_csv(file_name):
     os.makedirs(f'{OUTPUT_DIR}/json',exist_ok=True)
     df = pd.read_json(f'{OUTPUT_DIR}/json/{file_name}.json').transpose()
-    df = df.applymap(convert_array_to_string)
+    df = df.map(convert_array_to_string)
```

Comment:
I'm mega confused. I'm getting an error on this forked repo `'DataFrame' object has no attribute 'map'.`. But on my main repo folder, it works totally fine. I won't worry about it for now, since csv is a super minor component, and might not be needed at all

Reply:
Yeah, it's a module issue.  I created #42 to fix, since it's a warning it doesn't need to be fixed when/if we update `pandas`